src/cryptoadvance/specter/helpers.py

```diff
@@ -307,3 +308,17 @@ def get_address_from_dict(data_dict):
     if addr and addr != "Fee":
         return addr
     raise RuntimeError(f"Missing address info in object {data_dict}")
+
+
+def robust_json_dumps(obj):
```

Comment:
Good point. json parsing is sometimes quite annoying. But can you move that to somewhere in util? `util/json_rpc.py` would do it or a new `util/misc.py`. At some point, all that stuff in the util folder was initially in `helpers.py` and hopefully at some point the `helpers.py` vanish.

Maybe you can add a deprecation warning like 
```
Additions to helpers.py are deprecated. Please place your helpers in utils/somewhere
```

Reply:
Done